checkpoints/download_models.py

```diff
@@ -0,0 +1,6 @@
+from huggingface_hub import hf_hub_download
+
+FILENAME_VELOCITY = "midi-transformer-2023-10-03-08-34.pt"
+FILENAME_DSTART = "midi-transformer-2023-10-06-11-55.pt"
+hf_hub_download(repo_id="wmatejuk/midi-velocity-transformer", filename=FILENAME_VELOCITY, local_dir="velocity")
+hf_hub_download(repo_id="wmatejuk/midi-dstart-transformer", filename=FILENAME_DSTART, local_dir="dstart")
```

Comment:
```suggestion
hf_hub_download(
    repo_id="wmatejuk/midi-velocity-transformer",
    filename=FILENAME_VELOCITY,
    local_dir="checkpoints/velocity",
    local_dir_use_symlinks=False,
)
hf_hub_download(
    repo_id="wmatejuk/midi-dstart-transformer", 
    filename=FILENAME_DSTART, 
    local_dir="checkpoints/dstart",
    local_dir_use_symlinks=False,
)
```

Right now this creates new `velocity` and `dstart` folders in the root folder of ther repo, and that's not where the dashboards expect them to be. 

Also, without setting `local_dir_use_symlinks=False` the file is downloaded to the HF cache and in the folder we only get a link which I think is less convenient.